specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1463 @@
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseV2"
+            },
+            "examples": {
+              "application/json": "{\r\n  \"error\": {\r\n    \"code\": \"InternalServerError\",\r\n    \"message\": \"Internal Server Error\",\r\n    \"target\": \"document\",\r\n    \"innerError\": {\r\n      \"code\": 500000,\r\n      \"error\": \"Internal Server Error\",\r\n      \"message\": \"Unexpected internal server error has occurred\"\r\n    }\r\n  }\r\n}"
+            }
+          },
+          "503": {
+            "description": "Server temporary unavailable",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseV2"
+            },
+            "examples": {
+              "application/json": "{\r\n  \"error\": {\r\n    \"code\": \"ServiceUnavailable\",\r\n    \"message\": \"Service is temporary unavailable\",\r\n    \"target\": \"document\",\r\n    \"innerError\": {\r\n      \"code\": 503000,\r\n      \"error\": \"ServiceTemporaryUnavailable\",\r\n      \"message\": \"Service is currently unavailable.  Please try again later\"\r\n    }\r\n  }\r\n}"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Batch Submit": {
+            "$ref": "./examples/batch.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Translation"
+        ],
+        "summary": "Returns the status of the documents of a translation batch operation",
+        "description": "Returns the status of the list of documents translation operation status in the subscription\r\n            \r\nThe documents are sorted by the document Id\r\n            \r\nIf the number of documents exceed our paging limit, server side paging will be used.\r\nPaginated responses will indicate a partial result by including a continuation token in the response. The absence of a continuation token means that no additional pages are available.\r\n            \r\nClients MAY use $top and $skip query parameters to specify a number of results to return and an offset into the collection.\r\nThe server will honor the values specified by the client; however, clients MUST be prepared to handle responses that contain a different page size or contain a continuation token.\r\nWhen both $top and $skip are given by a client, the server SHOULD first apply $skip and then $top on the collection.\r\nNote: If the server can't honor $top and/or $skip, the server MUST return an error to the client informing about it instead of just ignoring the query options. This will avoid the risk of the client making assumptions about the data returned.",
```

Comment:
The canonical description (based on the path) would be to "list documents" since you are doing a get on the corresponding document collection URL. And the canonical pattern for the response model type name would be DocumentsList. Which is a paged response (with a value array with item type Document and a next link). 

I think we can simplify the naming of the concepts here.